### PR TITLE
Add link to ERD

### PR DESCRIPTION
**Pull request by TedLivist**
### In this project:
- We forked the Microverse ROR-Social-Scaffold repo
- We added the link to our project's Entity-Relations Diagram (ERD)
- The link is placed in the `docs/erd.html` file

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello Team 👋

**You've done pretty well here. Good ERD diagram. 👌**

However, let's make a couple of changes.

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: docs/erd.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>ERD Diagram Link</title>
+</head>
+<body>
+  <a target="_blank" href="https://lucid.app/publicSegments/view/52bedc68-9d04-48e8-98d4-4f0ea2d5ddd7/image.png">Link to ERD Diagram</a>
+</body>
```
Comment (mikenath223) [inline docs/erd.html:11]: - [ ] The Friend table isn't needed.
- [ ] Kindly add Primary keys(PK) and Foreign keys(FK) to tables, PK is however optional in the Friendship table.
- [ ] Take special note of the entity relationship, you keep mapping many to many. Kindly use the right relationship between entities.
- [ ] There should be a total of 5 tables; Post, Comment, and Likes tables should be added.

**Review by george-shammar — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUIRED** :recycle: :recycle: :construction_worker_man: 

Nice effort, Team.

**Highlight**

- [x] You've implemented some of the required changes from the previous review.

**Changes required**
Furtherance to the last review, please make the remaining changes below:
- [x] Your Comment and Like tables have no Primary Keys. Please add PK: ID to both tables.
- [x] The relationship you've modeled is wrong. In most cases, you've inverted the relationship. E.g You illustrated in your ERD that a post has many users. Likewise for the comments and likes. I encourage you to review the relevant lessons again for improved understanding of the concepts. Also, the extra label "Key" you added to every column that is not PK or FK is not necessary.

[OPTIONAL]

- [x] Please enable Github Action for linters in your repo. I'm making this optional because it's not in the review trail but it is a very important requirement. 
- [ ] The User table should also have an image_link column. This is also optional for the same reason as above. I encourage you to implement this.

Please make the required changes and submit another CRR when you're ready.

If you need clarification or support, please drop your message here and ping me on slack.

Cheers and happy coding :smile:

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested , 

- [ ] You have not made any change requested by the previous CR. Please make the changes and request another review. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by TedLivist** [comment]
I have made the changes requested by the second reviewer @george-shammar

I did not make a commit initially because the change is on a link outside the project on lucidcharts

If you visit the docs/erd.html.erb  file and click the link there you would see the changes made there

**Review by oloomoses — CHANGES_REQUESTED** [review]
## Changes Required
**Hello team,
Well done on implementing some of the changes suggested in the previous reviews.
Your ERD is impressive :clap:
Below are some of the issues to fix :point_down:**

- [ ] The relationship between a User and friendship table should look like this:
u

Kindly implement the above changes before submitting another code review.
### Feel free to leave a comment :point_down: while mentioning me. I will be happy to reply
## HAPPY CODING 
_Slack: Oloo Moses_

**Review by oloomoses — APPROVED** [review]
## Congratulations, Milestone Approved. :muscle: :tada: :clap: 
**Hello team,
Great job working on the changes suggested.
You have passed all the requirements for this milestone :muscle:**

Feel free to merge and close this pull request.

**I wish you the best in your next milestone and see you at the top :rocket:** 
#### Happy Coding
_Slack: Oloo Moses_

**Pull request merged**